hack/update-codegen-dockerized.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+GOPATH=`go env GOPATH`
+ANTREA_PKG="github.com/vmware-tanzu/antrea"
+
+# Generate protobuf code for CNI gRPC service with protoc.
+protoc --go_out=plugins=grpc:. pkg/apis/cni/v1beta1/cni.proto
+
+
+# Generate mocks for testing with mockgen.
```

Comment:
wait, why did we go from a simple `go generate ./...` command that takes care of generating all the mocks to a list of individual `mockgen` invocations? I don't seen an explanation in your PR comment. I personally prefer the "old" way because it keeps the `mockgen` command next to the Go interface definition. Was there a drawback to that approach?

Reply:
It's to unify the style and scripts we invoke code generation commands. Since we don't use inline comments to generate code for the other two resources (CNI, CRD), I thought it might be better to use a single way to organize the commands and know what code we have generated, as it's a little not convenient to know it from inline comments while maintaining a doc to list all mocks seems not efficient.
Let me know if the explanation makes sense to you.

Reply:
I thought keeping the generation command next to the interface definition was a nice principle, but I agree that the documentation file in its current form is not optimal. It's not something I feel strongly about however...

maybe you would consider defining a function in `update-codegen-dockerized.sh` to avoid repeating the mockgen invokation, but again not something I feel strongly about.

Reply:
Nice suggestion, done.